Week1/homework/app.js

```diff
@@ -1,11 +1,109 @@
 'use strict';
-{
-  const bookTitles = [
-    // Replace with your own book titles
-    'harry_potter_chamber_secrets'
-  ];
 
+const bookTitles = {
+  court_of_the_stone_children: {
+    title: "Court of the Stone Children",
+    language: "English",
+    author: "Cameron, Eleanor",
 
-  // Replace with your own code
-  console.log(bookTitles);
+  },
+  alices_adventures_in_wonderland: {
+    title: "Alice's Adventures in Wonderland",
+    language: "English",
+    author: "Carroll, Lewis",
+
+  },
+  through_the_looking_glass: {
+    title: "Through the Looking Glass",
+    language: "English",
+    author: "Carroll, Lewis",
+
+  },
+  what_am_i: {
+    title: "What Am I?",
+    language: "English",
+    author: "Charles, N. N.",
+
+  },
+  the_city_of_gold_and_lead: {
+    title: "The City of Gold & Lead",
+    language: "English",
+    author: "Christopher, John",
+
+  },
+  the_pool_of_fire: {
+    title: "The Pool of Fire",
+    language: "English",
+    author: "Christopher, John",
+
+  },
+  the_white_mountains: {
+    title: "The White Mountains",
+    language: "English",
+    author: "Christopher, John",
+
+  },
+  the_dark_is_rising: {
+    title: "The Dark is Rising",
+    language: "English",
+    author: "Cooper, Susan",
+
+  },
+  charlie_and_the_chocolate_factory: {
+    title: "Charlie and the Chocolate Factory",
+    language: "English",
+    author: "Dahl, Roald",
+
+  },
+  james_and_the_giant_peach: {
+    title: "James and the Giant Peach",
+    language: "English",
+    author: "Dahl, Roald",
+
+  }
+};
+
+
+// Replace with your own code
+function createAndAppend(name, parent) {
+  const elm = document.createElement(name);
+  parent.appendChild(elm);
+  return elm;
+}
+function booksInformation() {
+  const list = document.createElement("section");
+  list.setAttribute("id", "bookList");
+  const ul = createAndAppend("ul", list);
+  document.body.appendChild(list);
+  for (let i = 0; i < Object.keys(bookTitles).length; i++) {
+    const myObject = Object.keys(bookTitles);
```

Comment:
`myObject` is a meaningful name for the writer while the reader is expecting a name that make sense for what `myObject` contains exactly.